<launch>
 
  <!-- BEGIN ROBOT 1-->
  <group ns="robot1">
    <!-- <param name="tf_prefix" value="robot1_tf" /> -->
    <include file="$(find multiple_turtlebots_nav)/launch/include/robot.launch.xml" >
      <arg name="initial_pose_x" value="0" />
      <arg name="initial_pose_y" value="0" />
      <arg name="initial_pose_z" value="0" />
      <arg name="initial_pose_yaw" value="-1.616458" />
      <arg name="robot_name"  value="robot1" />
    </include>
  </group>

  <!-- BEGIN ROBOT 2-->
  <group ns="robot2">
    <!-- <param name="tf_prefix" value="robot2_tf" /> -->
    <include file="$(find multiple_turtlebots_nav)/launch/include/robot.launch.xml" >
      <arg name="initial_pose_x" value="-1" />
      <arg name="initial_pose_y" value="1" />
      <arg name="initial_pose_z" value="0" />
      <arg name="initial_pose_yaw" value="-1.616458" />
      <arg name="robot_name"  value="robot2" />
    </include>
  </group>
</launch>